packages/autofix/lib/rules/no-unused-vars.js

```diff
@@ -25,6 +98,36 @@ module.exports = ruleComposer.mapReports(
             const grand = parent.parent;
 
             switch (parent.type) {
+                case "FunctionExpression":
+                case "FunctionDeclaration":
+                case "ArrowFunctionExpression":
+
+                    // Don't autofix unused functions, it's likely a mistake
+                    if (parent.id === node) {
+                        return null;
+                    }
+
+                    switch (config.args) {
+                        case "all": {
+                            const prefix = getIgnoredArgPrefix(config);
+
+                            if (prefix === null) {
+                                return null;
+                            }
+
+                            const i = node.range[0];
+
+                            return fixer.replaceTextRange([i, i], prefix);
+                        }
+                        case "after-used": {
+                            const comma = sourceCode.getTokenBefore(node, commaFilter);
+
+                            return [fixer.remove(comma), fixer.remove(node)];
+                        }
+                        default:
+                            throw new Error("Invalid rule config value for 'args'");
```

Comment:
nits: rule options will be validated before applied, so it seems not necessary to do such a thing. :)

Reply:
an eslint rule forced me to add a `default` case.